rskj-core/build.gradle

```diff
@@ -137,18 +194,15 @@ tasks.withType(Jar) {
     reproducibleFileOrder = true
     dirMode = 0775
     fileMode = 0664
```

Comment:
Can you move this to the `tasks.withType(AbstractArchiveTask)` block above in line 51, instead of setting it just for `Jar`? In that case, we can get rid of this `tasks.withType(Jar)` block altogether, as `Jar` inherits from `AbstractArchiveTask`. Also, I did a build without this set, and only 2 files those modes while every other file and directory has group write permission disabled, so I feel more inclined to set

```
dirMode = 0755
fileMode = 0644
```

instead.

Reply:
done